pkg/pop/poptokenutils.go

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+package pop
```

Comment:
Let's rename this file from `poptokenutils.go` to `msal.go` ?

Reply:
Renamed